plugins/github/go.mod

```diff
@@ -1,18 +1,122 @@
 module github.com/falcosecurity/plugins/plugins/github
 
-go 1.15
+go 1.22
+
+toolchain go1.22.4
```

Comment:
Can you provide context on the needs for this?

Reply:
Introduced by mistake, sorry.
Fixed.